objects/scienceoutpost/scienceinfobooth/scienceinfobooth.object

```diff
@@ -0,0 +1,91 @@
+      "direction" : "left",
+      "flipImages" : true,
+
+      "imagePosition" : [0, 0],
+      "frames" : 10,
+      "animationCycle" : 3.2,
+
+      "spaceScan" : 0.1,
+      "interactiveSpaces": [
+        [2, 2], [3, 2], [4, 2],
+        [2, 3], [3, 3], [4, 3]
+      ],
+      "anchors" : [ "bottom" ]
+    },
+    {
+      "imageLayers" : [ { "image" : "scienceinfobooth.png:idle", "fullbright" : true }, { "image" : "scienceinfoboothlit.png:idle" } ],
+      "direction" : "right",
+
+      "imagePosition" : [0, 0],
+      "frames" : 10,
+      "animationCycle" : 3.2,
+
+      "spaceScan" : 0.1,
+      "interactiveSpaces": [
+        [1, 2], [2, 2], [3, 2],
+        [1, 3], [2, 3], [3, 3]
+      ],
+      "anchors" : [ "bottom" ]
+    }
+  ],
+
+  "soundEffect" : "/sfx/objects/hubbub.ogg",
+
+  "animation" : "scienceinfobooth.animation",
+  "animationPosition" : [0, 0],
+  "animationParts" : {
+    "booth" : "scienceinfoboothlit.png",
+    "sign" : "scienceinfobooth.png"
+  },
+  "forceFinishAnimationsInInit" : false,
+
+  "scripts" : ["scienceinfobooth.lua"],
+  "scriptDelta" : 10,
+
+  "detectArea" : [ [-8, 0], [14, 6] ],
+
+  "mouthPosition" : [-1.25, -3],
+  "chatOptions" : [
+    "Welcome to the Science Outpost! Any friend of Vinalisj is a friend of ours. If you need some advice, I'm here! Just... try to stay away from Kevin.",
+	"First things first, I'd recommend you take a look at the Frackin' Universe guide codexes - you should have a set of them in your ship's locker, but just in case, there's also a set of them in a bookshelf near the west side of this building.",
+	"Some of the information in those codexes might not seem relevant right now, but even some of our veteran researchers have learned something they didn't know through those books, so keep them in mind!",
+	"Looks like most of your Protectorate crafting databanks have been scrambled at some point - You'll need to use your Tricorder to research how to craft a lot of things.",
+	"I'd recommend researching extraction technology sooner rather than later. A Hand Mill might not be as fast as a full Extraction Lab, but it's still one of the most powerful tools in your arsenal.",
+    "Having trouble keeping your ship's tank topped up? Remember, Erchius isn't the only fuel you can use - Radioactive isotopes, properly processed algae, and even pure Hydrogen can all act as fuel sources - and some are more effective than others!",
+    "Having trouble gaining research? Your passive research gain will increase the longer you play, and will also increase based on the tier of the planet you're on - Exploring more dangerous planets lets you gather research more quickly!",
+    "Okay, so a couple of coworkers were saying they were leaving to go snarsh some garflipps, and I asked them what that meant, and they looked at me like I was an idiot. Is that normal? Am I the wierd one for not knowing what that means?",
```

Comment:
Does it "spoil the joke" to have a character not know what snarshing garflipps is? I kind of like the idea that the player first sees the term here and assumes that it's a running nonsensical joke, which makes the eventual appearance of the Snarshing table hit even harder.